### ISSUE TITLE
Need a logo

### ISSUE THREAD

**Issue by nsoft** [nsoft/jesterj#164]
Problem is I have no artistic talent (see below). If someone would like to contribute something based on the image below that didn't make people's eyes bleed quite so badly (maybe with some color), that would be very welcome. As with any contribution, **Visual works (images, sketches, vector graphics etc) attached to this ticket should be are an implicit grant of full Ownership, Copyright & Unrestricted Exclusive License to reuse/reproduce the original or derivative works. This grant is made to to Needham Software LLC** (which holds the rights to JesterJ at the moment). Of course, if selected for use as or in the project logo, the final form will be released under an appropriate open source license. 

This is necessary so that the project can arrive at a logo that can be trademarked and protect that logo from use by other entities. Note that the Apache Software Foundation works also works hard to protect it's logos and trademarks (https://www.apache.org/foundation/marks/list/), there is nothing anti-open source about this.

![20220728_192729](https://user-images.githubusercontent.com/3340874/181654004-13b07cb7-fcc0-4f89-a296-be22ff0f25a9.jpeg)
!

**Comment by kwatters** [comment]
Fun... I'm not much of a graphic artist.. We did use 99 Designs with good success before.   https://99designs.com/

**Comment by nsoft** [comment]
Yeah, but they probably want to be paid for their work :). Also, it is nice if the art comes from the community. Gives folks with additional talents a way contribute. This may take a while but that's ok.

**Comment by nsoft** [comment]
Ok well 2.5 years is long enough for this to remain open, so https://99designs.com/logo-design/contests/logo-jesterj-open-source-project-1345657/brief

**Comment by nsoft** [comment]
https://99designs.com/contests/poll/2b3136650a - express your preference here before next monday

**Comment by nsoft** [comment]
Logo selected, uploaded, front page readme updated :)
